### PR TITLE
feat(org-stats): Set up feature-flag, types, and page

### PR DESCRIPTION
**Pull request by leedongwei**

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationStats/redesign/index.tsx [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+      });
+    }, 3000);
+  }
+
+  /**
+   * Fetches stats of projects that the user has access to
+   */
+  getProjectsStats() {
+    return [];
+  }
```
Comment (leedongwei) [inline src/sentry/static/sentry/app/views/organizationStats/redesign/index.tsx:105]: Placeholder until [ER-605](https://getsentry.atlassian.net/browse/ER-605) is completed.

**Review by dashed — APPROVED** [review]

File: src/sentry/static/sentry/app/views/organizationStats/container.tsx [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+    });
+
+    Object.keys(this.state.rawProjectData).forEach(statName => {
+      this.props.api.request(statEndpoint, {
+        query: {
+          since: this.state.querySince,
+          until: this.state.queryUntil,
+          stat: statName,
+          group: 'project',
+        },
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/organizationStats/container.tsx:185]: Will we be adding new endpoints that can allow the client to be less chatty with the server?

File: src/sentry/static/sentry/app/views/organizationStats/redesign/index.tsx [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+  setSelectedDataCategory = (dataCategory: DataCategory) => {
+    this.setState({dataCategory});
+  };
+  get selectedDataCategory() {
+    return capitalize(this.state.dataCategory);
+  }
+
+  /**
+   * Fetches aggregated stats of tne entire organization
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/organizationStats/redesign/index.tsx:64]:
```suggestion
   * Fetches aggregated stats of the entire organization
```

File: src/sentry/static/sentry/app/views/organizationStats/container.tsx (hunk 2 above)
Comment (JoshFerge) [reply src/sentry/static/sentry/app/views/organizationStats/container.tsx:185]: yes, the page will only need to make two requests per time filter -- one to get overall org stats and one to get stats for projects the user belongs to

File: src/sentry/static/sentry/app/views/organizationStats/redesign/index.tsx [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+  componentDidMount() {
+    this.getOrganizationStats();
+    this.getProjectsStats();
+  }
+
+  setSelectedDataCategory = (dataCategory: DataCategory) => {
+    this.setState({dataCategory});
+  };
+  get selectedDataCategory() {
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/organizationStats/redesign/index.tsx:59]:
```suggestion
  };
  
  get selectedDataCategory() {
```

File: src/sentry/static/sentry/app/views/organizationStats/redesign/index.tsx [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+  getOrganizationStats() {
+    this.setState({orgStatsLoading: true});
+
+    const orgStats: OrganizationUsageStats = {
+      statsErrors: [],
+      statsTransactions: [],
+      statsAttachments: [],
+    };
+
+    for (let i = 0; i < 31; i++) {
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/organizationStats/redesign/index.tsx:75]: please why 31?

File: src/sentry/static/sentry/app/views/organizationStats/redesign/index.tsx [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+      const stats = {
+        ts: i.toString(),
+        accepted: {timesSeen: i * 100, quantity: i * 1000},
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/organizationStats/redesign/index.tsx:78]: maybe we could create a const for `quantity` and `timesSeen` and just reuse?

File: src/sentry/static/sentry/app/views/organizationStats/redesign/index.tsx (hunk 5 above)
Comment (markstory) [reply src/sentry/static/sentry/app/views/organizationStats/redesign/index.tsx:75]: Longest calendar month is 31 days.

File: src/sentry/static/sentry/app/views/organizationStats/redesign/index.tsx (hunk 6 above)
Comment (leedongwei) [reply src/sentry/static/sentry/app/views/organizationStats/redesign/index.tsx:79]: This code would be replaced when the API is ready. It's behind a feature flag for now so I'm not planning to optimize it.

**Review by priscilawebdev — APPROVED** [review]
lgtm

**Pull request merged**